tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -86,6 +86,9 @@ static const char * const devices[] = {
 #ifdef CONFIG_COUNTER_XLNX_AXI_TIMER
 	LABELS_FOR_DT_COMPAT(xlnx_xps_timer_1_00_a)
 #endif
+#ifdef CONFIG_COUNTER_MCUX_CTIMER
+								LABELS_FOR_DT_COMPAT(nxp_lpc_ctimer)
```

Comment:
Please decrease the indentation to match the other entries.

Reply:
Fixed.